### PR TITLE
Add dashboard for Kubernetes pods

### PR DESCRIPTION
**Pull request by jeffkreeftmeijer**
Much like #64, but for pods. Part of https://github.com/appsignal/appsignal-kubernetes/pull/2.

### CODE REVIEW

**Comment by matsimitsu** [comment]
Any idea how many additional metrics this would generate, asking for a friend..

File: dashboards/kubernetes/README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 There is currently one automated dashboard for AppSignal for Kubernetes:
```
Comment (unflxw) [inline dashboards/kubernetes/README.md:3]:
```suggestion
The following automated dashboards exist for AppSignal for Kubernetes:
```

**Review by unflxw — APPROVED** [review]

**Comment by unflxw** [comment]
As mentioned in #64, it would be nice to have human readable descriptions.

**Review by thijsc — APPROVED** [review]

**Comment by matsimitsu** [comment]
> Hm, tell your friend we're not sure as of right now. In theory, a Kubernetes cluster could have a lot of pods, much like a non-Kubernetes setup could have a lot of hosts. Would not knowing this be a blocker to push this out?

Not nessicarily, it's mainly that we know lots of short-lived hosts(pods) could be an issue in terms of fetching the metrics. It's a weak point in the way the current custom metrics system works, that if you have one metric name (e.g. `pod_load_avg` and it has a huge number of individual tags (`pod-xxx`) that it could time out. 

Also in the ingestion side we have a 4-megabyte document limit for MongoDB, and we know that customers with a ton of short-lived unique hosts hit that limit regularly. (The way it currently works is that the entire 4+ megabytes will be discarded, creating a gap in _all_ the charts).

**Review by matsimitsu — APPROVED** [review]

**Comment by jeffkreeftmeijer** [comment]
@matsimitsu thanks for the explanation here. Let's keep an eye on this.

I'm merging this now. Could you ping me when this is deployed? :)

**Comment by matsimitsu** [comment]
@jeffkreeftmeijer it's been deployed!

**Pull request merged**